### PR TITLE
Add docs on how to make a new release

### LINKED ISSUE #424: write down how to do a release
conda,  pypi etc.
for starters just make docs/release_howto.rst

### PR DESCRIPTION
**Pull request by bhazelton**
fixes #424

### CODE REVIEW

**Comment by bhazelton** [comment]
@dannyjacobs can you look at this?

File: docs/references/making_release.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+How to make a new release on PyPI and conda-forge
+
+PyPI:
+1. iterate the version number
```
Comment (dannyjacobs) [inline docs/references/making_release.md]: we use a system which uses <epochal>.<regular>.<fix>
<epoch> = very large changes. might happen every couple of years. Currently on 1.0
<regular> = regular releases. goal is to do ~3 times per year
<fix> = small patches to fix issues. happen as needed

File: docs/references/making_release.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+2. make the distribution: python setup.py sdist
+3. upload to test site: twine upload --repository testpypi dist/*
+4. check that it looks good at https://test.pypi.org/
+5. upload to real site: twine upload --repository pypi dist/*
+
+Conda:
+1. Fork the feedstock repo
```
Comment (dannyjacobs) [inline docs/references/making_release.md]: ie fork pyuvdata to your personal github account?

File: docs/references/making_release.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+2. Make a new branch on your fork for the changes
+3. get the new SHA from pypi: Go to the PyPI file listing page, next to each file there's a little link labeled SHA256 that will copy the right value to your clipboard. Or you can just download the file yourself and run `openssl sha256` on it.
+4. update recipe/meta.yaml: minimally update the version, build & SHA (ff it’s a new version, reset the build to zero, otherwise bump the build number), generally review the whole file for things that should change. (If these are the only changes, a bot may make a PR that does all this for you. If there are other needed changes and the bot makes a PR without those changes, comment on the PR to stop it being merged without the other fixes.)
```
Comment (dannyjacobs) [inline docs/references/making_release.md]: I can't say that I follow this procedure but it probably makes sense at the coal face.

File: docs/references/making_release.md (hunk 1 above)
Comment (bhazelton) [reply docs/references/making_release.md]: good idea.

File: docs/references/making_release.md (hunk 2 above)
Comment (bhazelton) [reply docs/references/making_release.md]: yes, I'll add that clarification

File: docs/references/making_release.md (hunk 3 above)
Comment (bhazelton) [reply docs/references/making_release.md]: I tried to make it a little clearer.

**Comment by bhazelton** [comment]
@dannyjacobs Thanks for the review, I think it's better now.

**Pull request merged**